wpeverest/everest-forms
# Feature - Popup Forms

### All Submissions:



### Changes proposed in this Pull Request:
Previously There is no feature like Popup forms. This PR enhance this feature.

### How to test the changes in this Pull Request:

1.  Create a form and copy the shortcode.
2. Goto the page and paste the shortcode and add the attributes in shotcode like [everest_form id="448" type="popup-link" text="popup" size="default"] 
3. type=popup-link,popup-button,popup and size=default,medium,large.
4. View the page see if it works or not.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Popup Forms.
